content/en/blog/_posts/2022-TBD-storage-capacity-GA/index.md

```diff
@@ -0,0 +1,73 @@
+---
+layout: blog
+title: "Storage Capacity Tracking reaches GA in Kubernetes 1.24"
+date: 2022-12-31
+slug: storage-capacity-ga
+---
+
+ **Authors:** Patrick Ohly (Intel)
+
+The v1.24 release of Kubernetes brings _storage capacity tracking_ as a generally available feature.
+
+## Problems we are solving
+
+As explained in more detail in the [previous blog post about this
+feature](/blog/2021/04/14/local-storage-features-go-beta/), storage capacity
+tracking allows a CSI driver to publish information about remaining
+capacity. The kube-scheduler then uses that information to pick suitable nodes
+for a Pod when that Pod has volumes that still need to be provisioned.
+
+Without this information, a Pod may get stuck without ever being scheduled onto
+a suitable nodes because kube-scheduler has to choose blindly and always ends
+up picking a node for which the volume cannot be provisioned because the
+CSI driver does not have sufficient storage left.
```

Comment:
the CSI driver does not have sufficient storage left ->
the underlying storage system managed by the CSI driver does not have sufficient capacity left

Reply:
Done.